### ISSUE TITLE
Boards definitions 

### ISSUE THREAD

**Issue by pipikupi** [Bodmer/OpenWeather#33]
Hi

After update of boards definition from 1.0.4 to 1.0.6 there is no weather forecast. Came back to 1.0.4 and everything came back to normal. I'm using arduino IDE

**Comment by dheydeck** [comment]
I think that this is an ESP32 core issue rather than a problem with Bodmer's outstanding code. Having said that, I've been having problems browsing to another ESP32-based project with iOS-based browsers which had been working perfectly previously. Based on this issue, I checked my ESP32 boards version and it was also 1.0.6. Reverting back to 1.0.4 fixed the problem. If there's not an issue against the ESP32 Core/Boards, I'll look at raising one.

**Comment by pipikupi** [comment]
Fully agree. I didn't add that as an issue to fix anything because it works. Hopefully it might help someone to find a solution

**Comment by Bodmer** [comment]
I think this is the same problem as #29 which was fixed in March 2021.

Can you check if you are using the latest version of the library.

**Comment by dheydeck** [comment]
Thank you, Bodmer. I will rebuild both projects and feed back.

**Comment by dheydeck** [comment]
Sorry about my delay in feeding back. I compiled my currently installed version of the OpenWeather example under ESP32 boards version 1.0.4, and it worked fine. I upgraded boards to 1.0.6 and the example ceased to work.
I then upgraded the OpenWeather library to the current version (0.2.1) and recompiled the example, which issued the messages relating to the connection to the secure server and that the certificate was not checked, and the example worked once more.
Sorry about the confusion caused.
